vassal-doc/src/main/readme-referencemanual/ReferenceManual/ReturnToDeck.adoc

```diff
@@ -6,29 +6,33 @@
 '''''
 
 === Return to Deck
-
-[cols=",",]
-|===
-|The _Return to Deck_ is used to move a piece into a <<Deck.adoc#top,Deck>> when the designated key command is received or the right-click context menu item is selected.
+The _Return to Deck_ is used to move a piece into a <<Deck.adoc#top,Deck>> when the designated key command is received or the right-click context menu item is selected.
```

Comment:
Should either delete the initial "The" or add the word "trait" after "_Return to Deck_".